Source/GamePadConfig.cpp

```diff
@@ -0,0 +1,41 @@
+#include "AppConfig.h"
+#include "GamePadConfig.h"
+#include "PathUtils.h"
+
+#define PROFILE_PATH (L"Profiles")

```

Comment:
Please make this `profiles` to be consistent with everything else that's already in the Data Files folder.

Reply:
can do that.